### PR TITLE
Contact detail views improvements

### PR DESCRIPTION
**Pull request by phgross**
- Link mail addresses and URLs 
- Only display archived-info collapsible header if archived information exists.

@deiferni

### CODE REVIEW

File: opengever/contact/models/url.py [hunk 1]
```diff
@@ -17,3 +18,8 @@ class URL(Base):
     contact = relationship("Contact", back_populates="urls")
     label = Column(String(CONTENT_TITLE_LENGTH))
     url = Column(String(CONTENT_TITLE_LENGTH))
+
+    def s_url(self):
```
Comment (deiferni) [inline opengever/contact/models/url.py]: hmm we should find a better name here.

**Comment by deiferni** [comment]
Looks 👍 , as discussed there are changes pending, though

**Comment by phgross** [comment]
@deiferni @lukasgraf updated ...

**Comment by phgross** [comment]
@lukasgraf rebased

**Comment by lukasgraf** [comment]
:+1:

**Pull request merged**